data/json/mutations/mutations.json

```diff
@@ -468,7 +468,7 @@
     "points": -1,
     "consume_time_modifier": 1.5,
     "description": "You've been taught proper table manners from your early childhood on; now, you can't even think about eating without being seated at a table and taking your time.  Eating without a table and chair frustrates you, but eating like a civilized person gives you a bigger morale bonus.",
-    "starting_trait": true,
+    "starting_trait": false,
```

Comment:
```suggestion
```
Mutations are default false for starting_trait, you can delete the line & do the same effect.
(Applies to the rest of the hobbified mutations, obv.)